Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -135,6 +140,12 @@ buildOnUpdateMessage req@RideCompletedBuildReq {} = do
             computed_value = fare
           }
       breakup = mkBreakupList (OnUpdate.BreakupPrice currency . fromIntegral) OnUpdate.BreakupItem req.fareParams
+      buildStartLocation = case req.startLocation of
+        Nothing -> Nothing
+        Just location -> Just $ buildLocation location
```

Comment:
Can be refactored, like
`buildStartLocation <- forM req.startLocation buildLocation`